client/src/components/Avatar.jsx

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
 
+const imgAvatar = require('../assets/images/avatars/2.png');
+
+
 export default function Avatar(props) {
     let avatarImage = 
         <div className='avatarDiv'>
-            <img className='avatarImage' src={require('../media/avatar.png')} alt='profile avatar'></img>
+            <img className='avatarImage' src={props.avatarUrl || imgAvatar} alt='profile avatar'></img>
```

Comment:
I think you meant 
src={props.avatar || imgAvatar}
and in this case lines 11 to 16 can be deleted
Am I right ?